R/series__series.R

```diff
@@ -159,6 +159,8 @@ Series_shape = method_as_active_binding(\() .pr$Series$shape(self))
 add_expr_methods_to_series = function() {
   methods_exclude = c(
     "agg_groups",
+    "over",
+    "rolling",
```

Comment:
I have removed these as they are not directly related to this PR, but obviously they do not work on the Series.